UIMentoringSession/MenteesChallenges/Pets Main Card UI Chellenge/OleksandrKharchenko/OlKha_PetsViewController.xib

```diff
@@ -1,22 +1,73 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="13142" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="21701" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
+    <device id="retina6_12" orientation="portrait" appearance="light"/>
```

Comment:
Bottom label in the cell without bottom and trailing constraints bug. Fixed width is a wrong strategy here. It doesn't work with a lot of text 
![Screenshot 2023-04-17 at 14 09 39](https://user-images.githubusercontent.com/14887101/232467784-865a8a5b-f64f-4a9b-82cb-e624fcc294f0.png)